lib/game.rb

```diff
@@ -0,0 +1,144 @@
+require_relative '../lib/player.rb'
+
+class Game
+  attr_reader :player1, :player2
+
+  def initialize(player1, player2, mark1, mark2)
+    @player1 = player1
+    @player2 = player2
+    @number_options = %w[1 2 3 4 5 6 7 8 9]
+    @arr = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @game_status = true
+    @active_player = player1
+    @win = ''
+    @mark1 = mark1
+    @mark2 = mark2
+  end
+
+  def create_table
+    system('cls')
+    system('clear')
+    puts @table = "
```

Comment:
No `puts/print` and `gets.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- [ ] Please remove all `puts/print` statement from this file.

**IMPORTANT!** This is just one place where I am pointing this out. Please go through this file carefully and remove all `puts/print` statements.